test/extensions/filters/http/kill_request/crash_integration_test.cc

```diff
@@ -0,0 +1,73 @@
+#include "envoy/extensions/filters/http/kill_request/v3/kill_request.pb.h"
+
+#include "test/integration/http_protocol_integration.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+
```

Comment:
Nit: please follow standard test extension namespace conventions. Should be qualified with extension and also put inside an inner anonymous namespace.

Reply:
Whoops. Done.